src/session_impl.cpp

```diff
@@ -6198,41 +6200,59 @@ namespace aux {
 
 	void session_impl::set_local_download_rate_limit(int bytes_per_second)
 	{
+#if TORRENT_USE_INVARIANT_CHECKS
```

Comment:
all these should use the macro, to check the invariant at the end of the function too